plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -880,6 +880,14 @@ def _update_location(package_element, filename):
         location_element = package_element.find('location')
         location_element.set('href', filename)
 
+    def get_symlink_name(self):
+        """
+        Provides the name that should be used when creating a symlink.
+        :return: file name as it appears in a published repository
```

Comment:
There should be a whitespace between the `:return: and the docstring description. (I think)